examples/active.ini

```diff
@@ -1,10 +1,11 @@
 [acq_params]
-amplitude = 1.0
-p90_us = 4.8
+amplitude = 0.575
+p90_us = 1.7
 deadtime_us = 10.0
 tau_us = 3500.0
 deblank_us = 1.0
 acq_time_ms = 1024.0
+sw_khz = 3.8999532005615936
```

Comment:
There is no default for this parameter so it needs to be defined in the active ini

Reply:
the NMR signal GUI also needs 'adc_offset' defined, should that also be included?

Reply:
I don't think that's true?

The point is that you have to press the button that determines the ADC offset before trying to acquire signal.

please reply here to confirm this, so that we can can resolve this

Reply:
so, what I'm saying is that `adc_offset` actually *should not* be in this file, b/c if you try to acquire NMR without first clicking the ADC offset button, the NMR GUI *should* fail.

Reply:
yes JF is correct if the GUI or anything is run without the adc_offset being set (which is set by running adc_offset.py) an error is thrown saying you skipped a step and need to determine adc_offset

Reply:
I'm going to go ahead and merge this PR.. please make sure that @wfkincaid is on the same page with this issue